src/overtone/helpers/synth.clj

```diff
@@ -10,9 +10,14 @@
   (when-not (valid-synth-node-pos pos)
     (throw (IllegalArgumentException. (str "Invalid synth node target. Was expecting one of " valid-synth-node-pos ", found: " pos)))))
 
+(def ^:dynamic *target-pos* nil)
```

Comment:
this is global scoped, could it be better to create an atom inside extract-target-pos-args* body?

Reply:
Oops, I will remove this tonight, some leftovers from an experiment I was doing, thanks o/